src/services/study/network-modifications.ts

```diff
@@ -189,7 +244,13 @@ export function generationDispatch(
     });
 }
 
-export function generatorScaling(studyUuid, nodeUuid, modificationUuid, variationType, variations) {
+export function generatorScaling(
+    studyUuid: UUID,
+    nodeUuid: UUID,
+    modificationUuid: UUID | undefined,
+    variationType: string,
+    variations: any[]
```

Comment:
any

Reply:
I couldn't find the variations type that's why I kept any as type.